docs/apache-airflow-providers-apprise/index.rst

```diff
@@ -0,0 +1,77 @@
+
+ .. Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+ ..   http://www.apache.org/licenses/LICENSE-2.0
+
+ .. Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+``apache-airflow-providers-apprise``
+======================================
+
+Content
+-------
+
+.. toctree::
+    :maxdepth: 1
+    :caption: Guides
+
+    Connection types <connections>
+
+
+.. toctree::
+    :maxdepth: 1
+    :caption: Commits
+
+    Detailed list of commits <commits>
+
+
+.. toctree::
+    :maxdepth: 1
+    :caption: Resources
+
+    Python API <_api/airflow/providers/apprise/index>
+    PyPI Repository <https://pypi.org/project/apache-airflow-providers-apprise/>
+    Installing from sources <installing-providers-from-sources>
```

Comment:
There is a missing link to the notifier hotow
I don't know why/how the doc build is missing it but this will cause failures when we try to release the provider. Like we had for https://github.com/apache/airflow/pull/32006

@potiuk this is interesting case that I can't explain. I would have expected the doc build to fail on this PR

Reply:
To be honest - I have no idea either. The errors that we had are generated by Sphinx and  theorethically it should have been complaining about missing index - but it does not. I don't even have hypothesis why it's doing that

Reply:
@eladkal @potiuk Added missing link for Apprise Notification.

Reply:
@eladkal @eladkal Just noticed we already have a link to `notifications/apprise_notifier_howto_guide` in line 79. Do we still need to mention this explicitly?

In [32006](https://github.com/apache/airflow/pull/32006) I guess it was missing this altogether, right?

Reply:
Ah.. That would explain why it worked :)   -  I think it's better to move it to the toc tree above from the end @utkarsharma2 - as it might confuse things and also when docs is generated ? In most other provider toc trees are in the same part of .rst file.

Reply:
@potiuk Yup, I agree sorry didn't check in with other providers earlier. Updated it. :)